bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/RetryOptions.java

```diff
@@ -350,7 +350,17 @@ public boolean isRetryable(Status.Code code) {
    * @return a {@link com.google.api.client.util.BackOff} object.
    */
   public BackOff createBackoff() {
-    return createBackoffBuilder().build();
+    return createBackoff(getMaxElapsedBackoffMillis());
+  }
+
+  /**
+   * <p>createBackoff.</p>
```

Comment:
Could you update the doc to explain when do we pass in maxElapsedBackoffMillis and when should not?